src/contract_client.rs

```diff
@@ -292,7 +301,7 @@ impl<Type> ContractInitHandle<Type> {
             crate::types::BlockItemSummaryDetails::AccountTransaction(at) => match at.effects {
                 AccountTransactionEffects::ContractInitialized { data } => {
                     let contract_client =
-                        ContractClient::new(self.client, data.address, data.init_name);
+                        ContractClient::new(self.client, data.address, data.init_name, None);
```

Comment:
Would it make sense to try to get the schema from the contract module here?

Could do
```suggestion
                        ContractClient::create(self.client, data.address)?;
```